tools/fetch.py

```diff
@@ -180,18 +192,25 @@ def vx_start(nodelist, outdir, cmd, user, timeout=0, threadcount=20):
             auto_script_post = "scripts/" + config.script + "-post.sh"
             f = open(auto_script, 'r')
         cmd_str = f.read()
-        auto_outdir = "logs/" + config.script.split(".")[0]
+        suffix = time.strftime('-%Y-%m-%dT%H:%M:%S', time.gmtime(time.time()))
+        auto_outdir = 'logs/' + config.script.split('.')[0] + suffix
+        latest_symlink = 'logs/' + config.script.split('.')[0] + '-latest'
```

Comment:
Could this line be simplified to the following to make things more flexible?  If in the future you change something about the auto_outdir you won't have to worry about also updating the the code for latest_symlink:

latest_symlink = auto_outdir + '-latest'